.storybook/preview.js

```diff
@@ -1,14 +1,16 @@
+import '../src/app/globals.css'
+
 /** @type { import('@storybook/react').Preview } */
 const preview = {
   parameters: {
-    actions: { argTypesRegex: "^on[A-Z].*" },
+    actions: { argTypesRegex: '^on[A-Z].*' },
```

Comment:
linter?

Reply:
Yeah linter - we need to set up some eslint/linting rules.